cmd/xl-v1-healing.go

```diff
@@ -28,7 +28,7 @@ func healFormatXL(storageDisks []StorageAPI) (err error) {
 	// Attempt to load all `format.json`.
 	formatConfigs, sErrs := loadAllFormats(storageDisks)
 
-	// Generic format check validates
+	// Generic format check checks
```

Comment:
This change has made the comment more cryptic - I suggest: `Generic format check steps:`